fix host variable when rsh is renamed to other.

Upstream-Status: Pending

If rsh is renamed to other, like rsh.netkit, host variable is assigned to
rsh.netkit, which is wrong.

Signed-off-by: Roy.Li <rongqing.li@windriver.com>
---
 rsh/rsh.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/rsh/rsh.c b/rsh/rsh.c
index ac594f9..11f533e 100644
--- a/rsh/rsh.c
+++ b/rsh/rsh.c
@@ -100,7 +100,6 @@ main(int argc, char *argv[])
 #else
 	if (!strcmp(p, "rsh")) asrsh = 1;
 #endif
-	else host = p;
 
 	/* handle "rsh host flags" */
 	if (!host && argc > 2 && argv[1][0] != '-') {
-- 
1.7.5.4

